api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java

```diff
@@ -158,9 +160,13 @@ public CohortDefinition createInProgram(String name, Program program) {
 	 * @return CohortDefinition
 	 */
 	public CohortDefinition getBaseCohort() {
-		return generalSql("baseCohort",
-		    BaseQueries.getBaseCohortQuery(hivMetadata.getARVAdultInitialEncounterType().getEncounterTypeId(),
-		        hivMetadata.getARVPediatriaInitialEncounterType().getEncounterTypeId(), hivMetadata.getHIVCareProgram().getProgramId(),
-		        hivMetadata.getARTProgram().getProgramId()));
+		Map<String, String> parameters = new HashMap<>();
```

Comment:
`new HashMap<>()` needs to have types defined - this prevents the module from loading successfully.

Reply:
This has been fixed